docker-compose-test.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.5'
+services:
+  test-db:
```

Comment:
Why do we need a database container to run the tests?

Reply:
Tests use database. It's better to use mysql rather than sqlite because there may be differences in how they work.